src/realm/realmReducer.js

```diff
@@ -72,6 +73,8 @@ export default (state: RealmState = initialState, action: Action): RealmState =>
         twentyFourHourTime: action.data.twenty_four_hour_time,
         canCreateStreams: action.data.can_create_streams,
         isAdmin: action.data.is_admin,
+
+        isTopicMandatory: action.data.realm_mandatory_topics,
```

Comment:
nit: Instead of starting a new group with an empty line, this could maybe be moved up to join the group that goes from `nonActiveUsers` to `videoChatProvider`. I'm just basing that on the fact that those use things from `action.data` that begin with `realm_`.

Reply:
done